### PR TITLE
Add tagging to cluster claiming

### PR DESCRIPTION
**Pull request by rayashworth**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,30 +14,49 @@ This repo represents the Community Automation effort where teams can contribute
...
 **NOTES:**
 
-- On RHEL8 use podman
 - You may need to do a **docker logout** before you begin.
 - default ssh keys are comming from your ~/.ssh/id_rsa and ~/.ssh/id_rsa.pub
 
 **Tested on MAC (big sur), Ubunutu 16.04,18.04,20.04, and RHEL 7/8**
 
+## MAC, Ubuntu, and RHEL 7 users
+
```
Comment (wkrapohl) [inline README.md:27]: I think what your doing here is creating a local environment to run in, might want to state that this is what is being created with this option.

File: README.md [hunk 2]
```diff
@@ -14,30 +14,49 @@ This repo represents the Community Automation effort where teams can contribute
...
+```
+# docker run -v /YOUR_REPO_PATH:/community-automation -v ~/.ssh:/root/.ssh -i -t quay.io/rayashworth/community-ansible:latest
```
Comment (wkrapohl) [inline README.md:29]: docker run -v /<YOUR_REPO_PATH>:/community-automation -v ~/.ssh:/root/.ssh -i -t quay.io/rayashworth/community-ansible:latest

File: README.md [hunk 3]
```diff
@@ -14,30 +14,49 @@ This repo represents the Community Automation effort where teams can contribute
...
+```
+
+### calling with playbook, assumes all vars are set in vars files
+
+```
+# docker run -v /YOUR_REPO_PATH:/community-automation -v ~/.ssh:/root/.ssh -i -t quay.io/rayashworth/community-ansible:latest -c "ansible-playbook -i YOUR-PLAY/inventory  YOUR_PLAYBOOK/playbook.yml"
```
Comment (wkrapohl) [inline README.md:35]: docker run -v /<YOUR_REPO_ABSOLUTE_PATH>:/community-automation -v ~/.ssh:/root/.ssh -i -t quay.io/rayashworth/community-ansible:latest -c "ansible-playbook -i <YOUR_PLAY_BOOK_ABSOLUTE_PATH>/inventory  <YOUR_PLAYBOOK_ABSOLUTE_PATH>/playbook.yml"

File: README.md [hunk 4]
```diff
@@ -14,30 +14,49 @@ This repo represents the Community Automation effort where teams can contribute
...
+```
+
+### using param passing, a mix of vars files, and command line vars
+
+```
+# docker run -v /YOUR_REPO_PATH:/community-automation -v ~/.ssh:/root/.ssh -i -t quay.io/rayashworth/community-ansible:latest -c "ansible-playbook -i YOUR-PLAY/inventory  YOUR_PLAYBOOK/playbook.yml -e \"var1=value1\" -e \"var2=value2\""
```
Comment (wkrapohl) [inline README.md:41]: docker run -v <YOUR_REPO_ABSOLUTE_PATH>:/community-automation -v ~/.ssh:/root/.ssh -i -t quay.io/rayashworth/community-ansible:latest -c "ansible-playbook -i <YOUR_PLAYBOOK_ABSOLUTE_PATH>/inventory  <YOUR_PLAYBOOK_ABSOLUTE_PATH>/playbook.yml -e \"var1=value1\" -e \"var2=value2\""

File: README.md [hunk 5]
```diff
@@ -14,30 +14,49 @@ This repo represents the Community Automation effort where teams can contribute
...
 ```
-# RHEL 7: sudo systemctl start docker; sudo yum install -y docker
-# docker run -v /YOUR_REPO_PATH:/community-automation -i -t quay.io/rayashworth/community-ansible:latest
 
-NOTE: RHEL 8
-# podman run -v /YOUR_REPO_PATH:/community-automation -i -t quay.io/rayashworth/community-ansible:latest
+## RHEL 8
+
+```
+# podman run -v /YOUR_REPO_PATH:/community-automation -v ~/.ssh:/root/.ssh -i -t quay.io/rayashworth/community-ansible:latest
```
Comment (wkrapohl) [inline README.md:47]: Might want to state this is to be used to create a  local environment to run in

podman run -v <YOUR_REPO_ABSOLUTE_PATH>:/community-automation -v ~/.ssh:/root/.ssh -i -t quay.io/rayashworth/community-ansible:latest

**Review by wkrapohl — COMMENTED** [review]
A few comments

File: README.md [hunk 6]
```diff
@@ -14,30 +14,49 @@ This repo represents the Community Automation effort where teams can contribute
...
+```
 
-# if you setup the parameters for a play ahead of time, you can directly call the playbook
+### calling with playbook, assumes all vars are set in vars files
 
-# docker run -v /YOUR_REPO_PATH:/community-automation -v ~/.ssh:/root/.ssh -i -t community-ansible:latest -c "ansible-playbook -i YOUR-PLAY/inventory  YOUR_PLAY/playbook.yml"
+```
+# podman run -v /YOUR_REPO_PATH:/community-automation -v ~/.ssh:/root/.ssh -i -t quay.io/rayashworth/community-ansible:latest -c "ansible-playbook -i YOUR-PLAY/inventory  YOUR_PLAYBOOK/playbook.yml"
```
Comment (wkrapohl) [inline README.md:53]: podman run -v <YOUR_REPO_ABSOLUTE_PATH>:/community-automation -v ~/.ssh:/root/.ssh -i -t quay.io/rayashworth/community-ansible:latest -c "ansible-playbook -i <YOUR_PLAYBOOK_ABSOLUTE_PATH>/inventory  <YOUR_PLAYBOOK_ABSOLUTE_PATH>/playbook.yml"

File: README.md [hunk 7]
```diff
@@ -14,30 +14,49 @@ This repo represents the Community Automation effort where teams can contribute
...
+```
 
-NOTE: RHEL 8
+### using param passing, a mix of vars files, and command line vars
 
-# podman run -v /YOUR_REPO_PATH:/community-automation -v ~/.ssh:/root/.ssh -i -t community-ansible:latest -c "ansible-playbook -i YOUR_PLAYBOOK/inventory  YOUR_PLAYBOOK/playbook.yml"
+```
+# podman run -v /YOUR_REPO_PATH:/community-automation -v ~/.ssh:/root/.ssh -i -t quay.io/rayashworth/community-ansible:latest -c "ansible-playbook -i YOUR-PLAY/inventory  YOUR_PLAYBOOK/playbook.yml -e \"var1=value1\" -e \"var2=value2\"
```
Comment (wkrapohl) [inline README.md:59]: podman run -v <YOUR_REPO_ABSOLUTE_PATH>:/community-automation -v ~/.ssh:/root/.ssh -i -t quay.io/rayashworth/community-ansible:latest -c "ansible-playbook -i <YOUR_PLAY_ABSOLUTE_PATH>/inventory  <YOUR_PLAYBOOK_ABSOLUTE_PATH>/playbook.yml -e \"var1=value1\" -e \"var2=value2\"

File: README.md (hunk 1 above)
Comment (rayashworth) [reply README.md:27]: done

File: README.md (hunk 2 above)
Comment (rayashworth) [reply README.md]: done

File: README.md (hunk 3 above)
Comment (rayashworth) [reply README.md]: done

File: README.md (hunk 5 above)
Comment (rayashworth) [reply README.md]: done

File: README.md (hunk 4 above)
Comment (rayashworth) [reply README.md]: done

**Review by wkrapohl — APPROVED** [review]
Looking good

**Pull request merged**